Trappist/src/Promact.Trappist.Core/Controllers/TestController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Promact.Trappist.DomainModel.Models.Test;
+using Promact.Trappist.Repository.Tests;
+using Promact.Trappist.Utility.Constants;
+
+namespace Promact.Trappist.Core.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/tests")]
+    public class TestsController : Controller
+    {
+        private ITestsRepository _test;
+        private IStringConstants _message;
```

Comment:
variable name _message is not proper for string constants. Give some meaningful name like _stringConstants. As string constants will not only contain messages but also other string literals.